projects/bp-gallery/src/components/views/geoguessr/GeoMap.tsx

```diff
@@ -51,22 +56,28 @@ const MyMarker = ({
   position,
   isPositionable,
   setPosition,
+  ref,
 }: {
   position: LatLng;
   isPositionable: boolean;
   setPosition: (pos: LatLng) => void;
+  ref?: any;
 }) => {
```

Comment:
I think this is better implemented using [`forwardRef`](https://react.dev/reference/react/forwardRef). Where do you need the ref though? I can't find any uses.

Reply:
We use custom ref props instead of forwardRef in a lot of places though, might create an extra issue for that

Reply:
Do you have any in mind? A quick global search for `ref:` and `ref?:` didn't lead to any results besides this one and some irrelevant matches in `APIConnector.tsx`.